shipstation/api.py

```diff
@@ -445,7 +446,7 @@ def fetch_orders(self, parameters={}):
 
         valid_parameters = {self.to_camel_case(key): value for key, value in parameters.items()}
 
-        self.get(
+        return self.get(
```

Comment:
I'm fine with this change in principle, but I'd ask that you update the docstring to reflect a return

Reply:
Updated the docstring's first line and added a Returns explanation in the style of the Args and Raises.